driver-core/src/main/com/mongodb/assertions/Assertions.java

```diff
@@ -119,6 +126,57 @@ public static void doesNotContainNull(final String name, final Collection<?> col
         }
     }
 
+    /**
+     * @param value A value to check.
+     * @param <T> The type of {@code value}.
+     * @return {@code null}.
+     * @throws AssertionError If {@code value} is not {@code null}.
+     */
+    @Nullable
+    public static <T> T assertNull(@Nullable final T value) throws AssertionError {
```

Comment:
None of the other methods throw checked exceptions. So these follow the existing convention?